### DISCUSSION TITLE
RetroArch build instructions (and offline RetroAchievements support)

### DISCUSSION THREAD

**Discussion by meleu** [OnionUI/Onion#494] [Ideas]
It would :exploding_head: mindblowing if we had [RetroAchievements](https://retroachievements.org) support in such a cute and portable gadget.

Yeah, RetroAchievements requires network, but I'd be interested in working in a way to make them available for offline usage. But first I need to know how RetroArch is built.

I saw [in the wiki](https://github.com/jimgraygit/Onion/wiki/7.-Onion-for-advanced-users) a link for a dropbox directory but, eh..., that doesn't seem aimed to developers/hackers/contributors.

Could you guys please share how you guys build RetroArch?

Instructions for contributors about how to build/install a custom image would also be awesome.

**Comment by zebramonkeylizard** [comment]
I would like to build Retroarch with FFMPEG enabled to support video playback out of the box.
